data-model/SDP-data-model-templates/manuscript-object_sdp-template.json

```diff
@@ -160,6 +251,7 @@
         "change": "string describing what change was made?"
       }
     ],
-    "created": "xsd:dateTime"
+    "created": "xsd:dateTime",
+    "delivery": ""
```

Comment:
Another field we should have from airtable -- which can also be used like we do currently when checking data on stage by batch number